R/util-checkSpec.R

```diff
@@ -91,7 +91,9 @@ CheckSpec <- function(lData, lSpec) {
   }
   if (length(missingCols) > 0) {
     cli::cli_alert_danger("Not all required columns in the spec are present in the data, missing columns are: {missingCols}")
-  } else {
+  } else if (length(lSpecDataFrames) > 0) {
     cli::cli_alert("All {length(allCols)} required columns in the spec are present in the data: {allCols}")
```

Comment:
```suggestion
    cli::cli_alert("All {length(allCols)} required column{?s} in the spec are present in the data: {allCols}")
```
{cli} can automatically pluralize based on whether the number before the "{?s}" is 1 or something else, so we might as well use it!